### PR TITLE
Add usage examples

### PR DESCRIPTION
**Pull request by Sammers21**
Usage examples and general details are useful for giving an idea of what the library is about.

### CODE REVIEW

**Comment by g4s8** [comment]
@0crat elect

File: README.md [hunk 1]
```diff
@@ -26,6 +26,28 @@ This is the dependency you need:
 Read the [Javadoc](http://www.javadoc.io/doc/com.artipie/asto)
 for more technical details.
 
+Create a `hello.txt` file with `"Hello World!"` content on file-system-based
```
Comment (yegor256) [inline README.md]: @Sammers21 maybe you can here an intro text, explaining what this `Storage` is for? A few sentences. Now this sounds too cold, without an intro.

Comment (Sammers21) [reply README.md]: That is a good idea. Just done

**Comment by 0crat** [comment]
This pull request [#38](https://github.com/artipie/asto/issues/38) is assigned to @iakunin[/z](https://www.0crat.com/u/iakunin), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/e7dd83dc-f588-4068-b56b-c71a58b9a7cc); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: README.md [hunk 2]
```diff
@@ -26,6 +40,28 @@ This is the dependency you need:
...
```
Comment (iakunin) [inline README.md]: @Sammers21 maybe it's better to prepend this paragraph with a `Examples of usage` header?

**Comment by Sammers21** [comment]
@iakunin, review the pull request again, please

**Comment by 0crat** [comment]
Order was finished: +15 point(s) just awarded to @iakunin[/z](https://www.0crat.com/u/iakunin)

**Comment by 0crat** [comment]
Job audit: performer @iakunin[/z](https://www.0crat.com/u/iakunin)[/z](https://www.0crat.com/u/iakunin) didn't make CR comments

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Pull request merged**